### ISSUE TITLE
Transmission-rpc version issues in latest docker image

### ISSUE THREAD

**Issue by jimfinity** [Flexget/Flexget#3929]
Watchtower upgraded my flexget image to the latest docker release and the transmission plugin stopped working.

### Expected behaviour:

The docker image should have a transmission-rpc version that is compatible with flexget. According to the docs this is 4.3.1 https://flexget.com/en/Plugins/transmission

### Steps to reproduce:
Install the flexget 3.11.12 docker image from ghcr.io/flexget/flexget sha256:f44bab35ee48c5293479ec92e6ee23f7a958d769186e39a4ac1221212cfb0986

Any transmission plugin tasks will return the following:
```
2024-01-27 13:21:13 CRITICAL transmission  xwt             transmission-rpc module version mismatch, requiring transmission-rpc>=4.1.4,<5.0.0
2024-01-27 13:21:13 WARNING  task          xwt             Aborting task (plugin: transmission)
```
And running `pip show transmission-rpc` returns `Version: 7.0.3`

#### Config:
```yaml
content_filter:                                                                 
  reject:                                                                       
    - '*TBS*'                                                                   
    - '*TNT*'                                                                   
    - '*HDTV*'                                                                                                                            
  require_mainfile: false                                                       
  strict: false                                                                 
headers:                                                                        
  Cookie: <cookie>                    
notify:                                                                         
  entries:                                                                      
    message: Task {{ task }} found {{title}}                                    
    title: New Episode of {{series_name}}                                       
    via:                                                                        
      - pushover:                                                               
          api_key: <key>                               
          user_key: <key>                              
priority: 1                                                                     
rss:                                                                            
  all_entries: false                                                            
  ascii: false                                                                  
  escape: false                                                                 
  group_links: false                                                            
  silent: false                                                                 
  url: https://xtremewrestlingtorrents.net/rss.php?feed=dl&cat=75,79,77,56,83   
series:                                                                         
  1080p:                                                                        
    - AEW Dynamite:                                                             
        alternate_name: All Elite Wrestling Dynamite                            
    - AEW Collision:                                                            
        alternate_name: All Elite Wrestling Collision                           
    - AEW Rampage:                                                              
        alternate_name: All Elite Wrestling Rampage                             
    - AEW Battle Of The Belts:                                                  
        alternate_name: All Elite Wrestling Battle Of The Belts                                
set:                                                                            
  path: /downloads/tv/{{ series_name }}                                         
  priority: 1                                                                   
template:                                                                       
  - wrestlingtv                                                                 
  - save                                                                        
transmission:                                                                   
  host: <host>                                                              
  password: <password>                                          
  port: 9091                                                                    
  username: torrents    
```
  
#### Log:
```
2024-01-27 13:21:13 CRITICAL transmission  xwt             transmission-rpc module version mismatch, requiring transmission-rpc>=4.1.4,<5.0.0
2024-01-27 13:21:13 WARNING  task          xwt             Aborting task (plugin: transmission)
```
</details>

### Additional information:

- FlexGet version: 3.11.12
- Python version: 3.11.7
- Installation method: Docker
- Using daemon (yes/no): yes

I've worked around this in the short term by running `pip install transmission-rpc==4.3.1` in the container so the plugin now works.

**Comment by ksurl** [comment]
A fix has been merged into develop. Should be fixed in the next release.

**Comment by ksurl** [comment]
Should be fixed in 3.11.13 now
